src/main/kotlin/no/nav/familie/ef/sak/api/gui/VedtaksbrevController.kt

```diff
@@ -16,20 +17,25 @@ import java.util.*
 class VedtaksbrevController(private val brevService: VedtaksbrevService,
                             private val tilgangService: TilgangService) {
 
+    private val logger = LoggerFactory.getLogger(this.javaClass)
     @PostMapping("/{behandlingId}/{brevMal}")
-    fun forhåndsvisBrevV2(@PathVariable behandlingId: UUID,
-                          @PathVariable brevMal: String,
-                          @RequestBody brevRequest: JsonNode): Ressurs<ByteArray> {
+    fun lagSaksbehandlerbrev(@PathVariable behandlingId: UUID,
+                             @PathVariable brevMal: String,
+                             @RequestBody brevRequest: JsonNode): Ressurs<ByteArray> {
         tilgangService.validerTilgangTilBehandling(behandlingId)
-
-        val respons = brevService.forhåndsvisBrev(behandlingId, brevRequest, brevMal)
-
-        return Ressurs.success(respons)
+        val data = brevService.lagSaksbehandlerBrev(behandlingId, brevRequest, brevMal)
+        return Ressurs.success(data)
     }
 
     @GetMapping("/{behandlingId}")
```

Comment:
Burde det være tydligere i APIet att denne lager ett beslutterbrev? Bruk av POST i stedet for GET? 
@PostMapping("/{behandlingId}")

Vet ikke om det trengs noe mer i URL'en, endepunkten er ju "Vedtaksbrev" sånn att det er ju vedtaksbrevet man oppretter.

Reply:
fixd